ITEST/itest.webstudio/test-resources/EPBDS-12765/01-malicious-request.resp

```diff
@@ -1,4 +1,4 @@
-HTTP/1.1 404 Not Found
+HTTP/1.1 400
```

Comment:
Why have it changed to 400?

Reply:
Nice question!
This was handled by Jetty. So our firewall setting in WebStudio was not reached.
https://www.eclipse.org/jetty/documentation/jetty-10/programming-guide/index.html#pg-server-compliance-http